docs/content/content/content-principles.mdx

```diff
@@ -0,0 +1,170 @@
+
+To ensure content is consistent, follow our [Content styles](./content-styles) and [Content patterns](./content-patterns). This way users will have a seamless end-to-end journey from one page to another.
+
+<BackToTop />
+
+## Accurate
+
+Content must be factually correct. Ensure you research content from credible sources and cross-check multiple sources of truth if you need to.
+
+Check content with any relevant subject matter experts and make sure they confirm its accuracy.
+
+<BackToTop />
+
+## Comprehensive
+
+Ensure your content covers everything needed to support the page’s primary aim. This might include links to other supporting content or to more details.
+
+But too much information can overwhelm users and increase their cognitive load. That is, provide more information than a user’s working memory can hold in the moment.
+
+If you think some users might want more detail than what’s necessary, use ‘progressive disclosure’. Progressive disclosure means providing content in stages.
+
+Offer essential content to the user first. Provide any details that are relevant – but not essential – next on the same page, or by linking to another.
+
+Users who just want to complete a task then have relevant content where they need it. But it still meets the needs of those who may want to know more.
+
+<BackToTop />
+
+## Contextual
+
+Let users know where they are on in the Export Service by providing site context, through breadcrumbs or other means.
+
+Ensure the user has access to the right content when they need it, at each stage of their journey. It should be the least amount of information they need to ‘do the thing’ in the Export Service.
+
+Make sure there are clear exit points and navigation to previous or next steps in the user journey. Link to related supporting content.
+
+<BackToTop />
+
+## Unique
+
+While content should be consistent across the Export Service, it should not be uniform or duplicated.
+
+Avoid using significant chunks of the same content across pages. It’s a bad user experience and may lower page rankings in search engine results.
+
+Content for each web page should be unique so users and search engines can clearly understand the difference between them and easily select a relevant page.
+
+If content already appears on the Export Service or the [Department of Agriculture, Fisheries and Forestry](https://www.agriculture.gov.au/) (DAFF) website, link to the authoritative source.
+
+<BackToTop />
+
+## Related links
+
+- [How to create guidance in the Export Service](/guides/how-to-create-guidance)
+- [Content structure](./content-structure)
+- [Content styles](./content-styles)
+- [Content patterns](./content-patterns)
+- [Design principles](https://design-system.agriculture.gov.au/foundations/design-principles)
+- [Writing and designing content I Style Manual](https://www.stylemanual.gov.au/writing-and-designing-content)
+
+---
+
+## Notes
+
+<Box color="muted" fontSize="xs">
```

Comment:
This seems weird... can we please discuss?

Reply:
<img width="722" alt="Screenshot 2023-06-28 at 2 04 05 pm" src="https://github.com/steelthreads/agds-next/assets/6265154/de6be6ad-d1a7-410b-9faf-5f80c4d9e5da">

It creates weird margins